cmd/cc-uploader/main.go

```diff
@@ -45,78 +50,16 @@ const (
 	communicationTimeout        = 30 * time.Second
 )
 
-// Global WaitGroup to track uploads
-var uploadWaitGroup sync.WaitGroup
-
-func main() {
-	runtime.GOMAXPROCS(runtime.NumCPU())
-	flag.Parse()
-
-	uploaderConfig, err := config.NewUploaderConfig(*configPath)
-	if err != nil {
-		panic(err.Error())
-	}
-
-	logger, reconfigurableSink := lagerflags.NewFromConfig("cc-uploader", uploaderConfig.LagerConfig)
-
-	initializeDropsonde(logger, uploaderConfig)
+var (
+	// Global WaitGroup to track uploads
+	uploadWaitGroup sync.WaitGroup
+)
 
+func createShutdownSignal() <-chan os.Signal {
```

Comment:
naming: this creates a channel that contains shutdown signals, not a shutdown signal.